Resources/Prototypes/_NF/Hydroponics/seeds.yml

```
@@ -173,3 +173,32 @@
       Min: 1
       Max: 10
       PotencyDivisor: 10
+
+- type: seed
+  id: monkanana
+  name: seeds-monkanana-name
+  noun: seeds-noun-seeds
+  displayName: seeds-monkanana-display-name
+  plantRsi: _NF/Objects/Specific/Hydroponics/monkanana.rsi
+  packetPrototype: MonkananaSeeds
+  productPrototypes:
+    - FoodMonkana
```

```suggestion
    - FoodMonkanana
```
Consistency in naming.